Dockerfile

```diff
@@ -19,7 +19,6 @@ ENV DISABLE_SPRING="true" \
 WORKDIR /opt/system/
 
 ADD . ./
-ADD config/examples/*.yml config/
```

Comment:
We need to keep this.

Reply:
No, `config/examples` doesn't exist anymore and its files are already under `config`